### PR TITLE
Updating git usage vingette

### PR DESCRIPTION
**Pull request by bms63**

### CODE REVIEW

**Comment by bms63** [comment]
@thomas-neitmann Is there any policy on embedding images into Articles.  I wanted to give a little more guidance on using Github instead of the terminal, but finding pictures are more ideal.

File: vignettes/git_usage.Rmd [hunk 1]
```diff
@@ -19,6 +19,10 @@ knitr::opts_chunk$set(
 
 admiral GitHub: https://github.com/Roche-GSK/admiral.git
 
+# Introduction
+
+This article will give you an overview of how the admiral project is utilizing the version-control software git and the website Github while working with RStudio.  We will go over the primary branches that house the source code for the admiral project as well as how we use **Feature** branches to address Issues.  Issues can range from bugs to enhancement that have been identified or requested by developers, users or testers.  Finally, we provide the bare minimum git commands needed to get up and running.  Please refer to the Resource section for more in-depth guidance on using git and Github.
```
Comment (thomas-neitmann) [inline vignettes/git_usage.Rmd:24]: Please write it as Git**H**ub.

File: vignettes/git_usage.Rmd [hunk 2]
```diff
@@ -62,21 +66,34 @@ You can also create a feature branch in Github.
...
+### Commits from the Terminal in RStudio
+
 When committing, prefix the message with the issue number and add a meaningful message  
 `git commit –m '#94 last alive date implementation'`
 
-**Pull request**  
+### Commits from the Git Tab in RStudio
+
+You can also make use of the Git Tab within RStudio to committ your changes.  A benefit of using this Tab is being able to see your changes to the file with red and green highligting.  
+
```
Comment (thomas-neitmann) [inline vignettes/git_usage.Rmd:77]: Typo: ~highligting~ -> highlighting

File: vignettes/git_usage.Rmd (hunk 1 above)
Comment (thomas-neitmann) [reply vignettes/git_usage.Rmd:24]: Also please always wrap git in backticks such that it rendes like `git`.

Comment (thomas-neitmann) [reply vignettes/git_usage.Rmd:24]: You seem to be using two spaces after a period which is contrary to the existing text. Please replace by only a single one for consistency.

File: vignettes/git_usage.Rmd [hunk 3]
```diff
@@ -103,15 +120,17 @@ git push
 
 ### Solving Merge Conflicts in Github
 
+A simple write up with an image here.
```
Comment (thomas-neitmann) [inline vignettes/git_usage.Rmd:123]: Looking forward to seeing that 😉

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

**Comment by bms63** [comment]
**Notable Updates**
* Discussion around GitHub now included - original was just about git
* Images included around GitHub discussion and best practices
* Short discussion on protected branches


image

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]
Great stuff @bms63 👏 

I left some more comments but these updates will be extremely helpful to anyone wanting to contribute to admiral.

File: vignettes/git_usage.Rmd [hunk 4]
```diff
@@ -17,56 +17,93 @@ knitr::opts_chunk$set(
 )
 ```
 
-admiral GitHub: https://github.com/Roche-GSK/admiral.git
+admiral GitHub: https://GitHub.com/Roche-GSK/admiral.git
+
+# Introduction
+
+This article will give you an overview of how the admiral project is utilizing the version-control software `git` and the website GitHub while working with RStudio. We will go over the primary branches that house the source code for the admiral project as well as how we use **Feature** branches to address **Issues.**  Issues can range from bugs to enhancement that have been identified or requested by developers, users or testers. Finally, we provide the bare minimum of `git` commands needed to get up and running. Please refer to the Resource section for more in-depth guidance on using `git` and GitHub.
```
[the remark this answers is not in the document]
Comment (bms63) [reply vignettes/git_usage.Rmd:24]: Updated

File: vignettes/git_usage.Rmd [hunk 5]
```diff
@@ -17,56 +17,93 @@ knitr::opts_chunk$set(
...
--	Pull the latest committed changes from the unmerged branch: `git pull`  
--	Check out your feature branch: `git checkout <my_branch>`  
--	Merge the unmerged branch to <my_branch>: `git merge <unmerged_branch>`
+### Create a New Feature Branch from GitHub (from `devel`)
+
+You can also create a feature branch in GitHub. 
+
+- Switch to the `devel` branch
+- Type in your new feature branch name
+- Click Create branch: 94_lstalvdt from `devel`
```
[the remark this answers is not in the document]
Comment (bms63) [reply vignettes/git_usage.Rmd:58]: Updated

File: vignettes/git_usage.Rmd [hunk 6]
```diff
@@ -17,56 +17,93 @@ knitr::opts_chunk$set(
...
+```{r, echo = FALSE}
+    knitr::include_graphics("GitHub_feature_branch.png", dpi = 144)
+```
+
+
+### Commits from the Terminal in RStudio
 
-**Commit**  
 When committing, prefix the message with the issue number and add a meaningful message  
 `git commit –m '#94 last alive date implementation'`
```
[the remark this answers is not in the document]
Comment (bms63) [reply vignettes/git_usage.Rmd:69]: Please have a look.  Trying to keep it short!

File: vignettes/git_usage.Rmd [hunk 7]
```diff
@@ -17,56 +17,93 @@ knitr::opts_chunk$set(
...
+**NOTE:**  Placing the issue number in your commit message allows new developers to quickly find discussion surrounding your issue. When pushed to GitHub the issue number will hyperlink to the issue tracker. A powerful tool for discussion.
+
+## Pull request
+
+We recommend a thorough read through of the articles, [Pull Request Review Guidance](pr_review_guidance.html) and the [Programming Strategy](programming_strategy.html) for in-depth discussions on a doing a proper Pull Request.  Pull Request authors will benefit from shorter review times by closely following the guidance provided in those two articles. Below we discuss some simple `git` commands in the terminal and on GitHub for doing a Pull Request. 
+
 Once all changes are committed, push the new branch to GitHub:  
 `git push -u origin <new_branch_name>`  
   
 In GitHub, under "Pull request", the user has to "Compare and pull request". The branch to merge with must be the `devel` branch (base = `devel`) and the compare branch is the new branch to merge. All committed changes are then available for review.
```
[the remark this answers is not in the document]
Comment (bms63) [reply vignettes/git_usage.Rmd:89]: New image of PR provided. 

 Is there a way to have like a 5 person team to be the reviewers for non-GSK/Roche people?  Might be handy to show users to add the reviewing team to the PR?  That way a bunch of people receive the notification.

File: vignettes/git_usage.Rmd [hunk 8]
```diff
@@ -55,49 +54,56 @@ You can also create a feature branch in GitHub.
...
 
 ## Pull request
 
-We recommend a thorough read through of the articles, [Pull Request Review Guidance](pr_review_guidance.html) and the [Programming Strategy](programming_strategy.html) for in-depth discussions on a doing a proper Pull Request.  Pull Request authors will benefit from shorter review times by closely following the guidance provided in those two articles. Below we discuss some simple `git` commands in the terminal and on GitHub for doing a Pull Request. 
+We recommend a thorough read through of the articles, [Pull Request Review Guidance](pr_review_guidance.html) and the [Programming Strategy](programming_strategy.html) for in-depth discussions on a doing a proper Pull Request.  Pull Request authors will benefit from shorter review times by closely following the guidance provided in those two articles. Below we discuss some simple `git` commands in the terminal and on GitHub for doing a Pull Request. We recommend doing the Pull Request in GitHub only and not through the terminal.
```
Comment (thomas-neitmann) [inline vignettes/git_usage.Rmd:84]: Can you even do pull request on the terminal? I thought that way an GitHub exclusive feature...

**Review by thomas-neitmann — APPROVED** [review]
Great stuff 👏 

Thanks a lot @bms63!

File: vignettes/git_usage.Rmd (hunk 7 above)
[the remark this answers is not in the document]
Comment (thomas-neitmann) [reply vignettes/git_usage.Rmd]: I think you can add up to 10 reviewers on a single PR.

File: vignettes/git_usage.Rmd (hunk 8 above)
Comment (bms63) [reply vignettes/git_usage.Rmd:84]: Honestly, I thought you could but have never tried.  https://tighten.co/blog/open-github-pull-request-from-terminal/

**Pull request merged**